build.sbt

```diff
@@ -7,15 +7,15 @@ import scalariform.formatter.preferences._
 
 name := """atomic-store"""
 
-version := "0.0.5"
+version := "0.0.6-SNAPSHOT"
 
 organization := "net.artsy"
 
 homepage := Some(url("https://github.com/artsy/atomic-store"))
 
 licenses +=("MIT", url("https://opensource.org/licenses/MIT"))
 
-scalaVersion := "2.11.8"
+scalaVersion := "2.11.11"
```

Comment:
This can definitely wait, but I wonder if we should be [cross-building](http://www.scala-sbt.org/0.13/docs/Cross-Build.html) for 2.12 too. It _seems_ pretty simple, but don't really know.